packages/lit-dev-content/site/docs/components/lifecycle.md

```diff
@@ -357,22 +357,79 @@ window.onunhandledrejection = function(e) {
 
 ### Implementing additional customization {#reactive-update-cycle-customizing}
 
-#### performUpdate()  {#performupdate}
+This section covers some less-common methods for customizing the update cycle.
 
-Implements the reactive update cycle, calling the other methods, like `shouldUpdate()`, `update()`, and `updated()`.
+#### scheduleUpdate() {#scheduleupdate}
 
-Call `performUpdate()` to immediately process a pending update. This should generally not be needed, but it can be done in rare cases when you need to update synchronously.
+Override `scheduleUpdate()` to customize the timing of the update cycle. `scheduleUpdate()` is called when you request an update, and by default it calls `performUpdate()` immediately. Override it to defer the update. For example, you could use it to coordinate update timing between multiple elements.
```

Comment:
I think this is a little tricky, because we don't talk about enqueueUpdate() ... We only talk about `requestUpdate()` and `performUpdate()`, and `scheduleUpdate()` comes right in the middle. Nevertheless, rephrased for now. If we re-do the diagrams, might add "enqueue update" as a fuzzy step between the "request and update" and "perform an update" diagrams.